e2e/kind-config.yaml

```diff
@@ -11,11 +12,5 @@ nodes:
     hostPort: 5001
     listenAddress: "0.0.0.0"
     protocol: tcp
-  - containerPort: 10000
```

Comment:
Why are we removing these port mappings?

Reply:
We weren't using them. The original intent was to curl the gameserver locally to see if the "fake game" web server was up, but this never worked since we created a lot of servers and we couldn't do port mapping for all of them. We could create a test Pod in the cluster to curl all of them and make sure they would work, though.

Reply:
added an issue to track this: https://github.com/PlayFab/thundernetes/issues/290